site/en/workflows.md

```diff
@@ -1319,10 +1458,7 @@ ocrd process \
   "tesserocr-deskew -I OCR-D-BIN-DENOISE -O OCR-D-BIN-DENOISE-DESKEW -P operation_level page" \
   "cis-ocropy-segment -I OCR-D-BIN-DENOISE-DESKEW -O OCR-D-SEG-REG -P level-of-operation page" \
   "tesserocr-deskew -I OCR-D-SEG-REG -O OCR-D-SEG-REG-DESKEW" \
-  "cis-ocropy-clip -I OCR-D-SEG-REG-DESKEW -O OCR-D-SEG-REG-DESKEW-CLIP" \
-  "cis-ocropy-segment -I OCR-D-SEG-REG-DESKEW-CLIP -O OCR-D-SEG-LINE" \
-  "cis-ocropy-clip -I OCR-D-SEG-LINE -O OCR-D-SEG-CLIP-LINE -P level-of-operation line" \
-  "cis-ocropy-dewarp -I OCR-D-SEG-CLIP-LINE -O OCR-D-SEG-LINE-RESEG-DEWARP" \
+  "cis-ocropy-dewarp -I OCR-D-SEG-REG-DESKEW -O OCR-D-SEG-LINE-RESEG-DEWARP" \
```

Comment:
This change does not work quite like that.

Why deskew on the region level after already segmenting lines? If you don't also segment lines again afterwards, the regions' orientation will be ignored. 

If you expect regions with 90° or 270°, then you need keep the ocrd-cis-ocropy-segment step in there.

So either just remove the clipping steps, or also remove the region-deskewing step.

Reply:
I am not even sure where this change comes from, wasn't a deliberate choice to change the existing recommendations (though we should). But thanks for the notes, I'll adapt the workflow.

Reply:
removed clipping and deskewing from the "best" workflow, the second binarization from the "fast" workflow.